### PR TITLE
fix: change the log message when the status is changed

### PR DESCRIPTION
**Pull request by yohanlaborda**
# [TR-3332](https://oat-sa.atlassian.net/browse/TR-3332)

Redirect to waiting page when test is paused

## Changelog
- fix: change the log message when the status is changed

### CODE REVIEW

**Review by augustas — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Review by poyuki — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

File: model/delivery/AutoStartProctoredDeliveryService.php [hunk 1]
```diff
@@ -91,9 +91,12 @@ private function changeStateToAuthorized(DeliveryExecution $deliveryExecution):
         $deliveryExecution->getImplementation()->setState(ProctoredDeliveryExecution::STATE_AUTHORIZED);
 
         $this->logger->info(
-            get_called_class() . ' changes the status of delivery execution to authorized',
+            sprintf(
+                'Changes in the status of delivery execution to authorized. Delivery Execution Identifier: %s',
```
Comment (wazelin) [inline model/delivery/AutoStartProctoredDeliveryService.php:95]: I fail to understand the message here.
Did you mean something like that?
```suggestion
                '"%s" Delivery Execution's status changed to authorized',
```

**Review by wazelin — CHANGES_REQUESTED** [review]

**Review by wazelin — APPROVED** [review]

**Review by wazelin — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Pull request merged**